docs/core/tools/dotnet-tool-list.md

```diff
@@ -0,0 +1,59 @@
+---
+title: dotnet tool list command - .NET Core CLI
+description: The dotnet tool list command lists the specified .NET Core Global Tool from your machine.
+author: mairaw
+ms.author: mairaw
+ms.date: 04/25/2018
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.workload: 
+  - dotnetcore
+---
+# dotnet tool list
+
+[!INCLUDE [topic-appliesto-net-core-2-1plus.md](../../../includes/topic-appliesto-net-core-2-1plus.md)]
+
+## Name
+
+`dotnet tool list` - Lists all [.NET Core Global Tools](global-tools.md) currently installed on your machine or specified path.
+
+## Synopsis
+
+```
+dotnet tool list <-g|--global>
+dotnet tool list <--tool-path>
+dotnet tool list [-h|--help]
+```
+
+## Description
+
+The `dotnet tool list` command provides a way for you to list all .NET Core Global Tools installed on your machine (current user profile) or specified path. The command lists the package name, version installed, and the global tool command. To use the list command, you either have to specify that you want to see all user-wide tools using the `--global` option or specify a custom path using the `--tool-path` option.
```

Comment:
same problem as above. Alternative resolution:

"list all .NET Core Global Tools installed user-wide on your machine (current user profile) or in the specified path